### DISCUSSION TITLE
Meeting this week (May 27 - Jun 2)

### DISCUSSION THREAD

**Discussion by furkan7258** [ud-turkic/udtw23#42] [General]
Dear all,

I thought we could meet this week after a while. Would you be able to meet this week and what times work for you? I thought just writing your available times here would be easier than a when2meet link. Please include your time zones as well. Thank you guys in advance!

Regards.

**Comment by coltekin** [comment]
The following times would work form me: 

- Wednesday, anytime before 12:00 CEST
- Thursday any time
- Except Friday, any day after 18:00 CEST

Weekend is also fine.

**Comment by Sudi2021** [comment]
For me:
- Thursday any time 
- Wednesday and Friday after 18:00 CEST
- Weekend any time

**Comment by BermetChontaeva** [comment]
For me: 

    - Thursday any time
    - Wednesday and Friday after 18:00 CEST
    - Weekend any time

**Comment by jonorthwash** [comment]
Easiest for me would be:
- Thursday between 13-14 CEST
- Friday 13-14, 15-17 CEST
- Saturday 14-16 CEST

None are ideal, and I could probably make it work other times, but maybe not?

**Comment by furkan7258** [comment]
Unfortunately, we haven't gotten many responses. What platform do you think would get responses from most people, maybe mail?

**Comment by furkan7258** [comment]
I've just created the following Discord channel, please consider joining [here](https://discord.gg/P24WxqAx) (expires in 7 days).
